src/Camera.cc

```diff
@@ -78,6 +78,12 @@ class sdf::Camera::Implementation
   /// \brief Far clip distance for depth camera.
   public: double depthFarClip{10.0};
 
+  /// \brief Segmentation type for segmentation camera.
+  public: std::string segmentationType;
+
+  /// \brief Boundingbox type for boundingbox camera.
+  public: std::string boundingBoxType;
```

Comment:
these variables should be initialized to their default values that were specified in `sdf/1.9/camera.sdf`